src/backend/distributed/commands/database.c

```diff
@@ -396,6 +441,20 @@ citus_internal_database_command(PG_FUNCTION_ARGS)
 			DropDatabase(NULL, (DropdbStmt *) parseTree);
 		}
 	}
+	else if (IsA(parseTree, AlterDatabaseStmt))
+	{
+		elog(DEBUG1, "Altering DB");
+		AlterDatabaseStmt *stmt = castNode(AlterDatabaseStmt, parseTree);
+
+		bool missingOk = false;
+		Oid databaseOid = get_database_oid(stmt->dbname, missingOk);
+
+
+		if (OidIsValid(databaseOid))
+		{
+			AlterDatabase(NULL, (AlterDatabaseStmt *) parseTree, true);
```

Comment:
Can you put `true` in a parameter?

Reply:
Removed totally since there is no need to use now